mocha-client-tests/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "mocha-test-webextension",
+  "version": "1.0.0",
+  "description": "Example how to run unit tests for WebExtension",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha-phantomjs -p ./node_modules/phantomjs-prebuilt/bin/phantomjs -R nyan tests/background.html",
+    "web-ext-run": "web-ext run  -s ./addon"
+  },
+  "author": "",
+  "license": "MPL-2.0",
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "chrome-mock": "0.0.9",
```

Comment:
chrome-mock used for phantomJS as it not have WebExtensions (or Chrome Extensions) API inside of engine, so i'm using mock of this API's. As when you include to test runner original file it throws errors about chrome is not defined